pep-0286.txt

```diff
@@ -5,110 +5,130 @@ Last-Modified: $Date$
 
-    In some cases, parsing an argument will allocate new memory, which
-    is then to be released by the caller.  This has two problems:
+In some cases, parsing an argument will allocate new memory, which
+is then to be released by the caller.  This has two problems:
 
-    1. In case of failure, the application cannot know what memory to
-       release; most callers don't even know that they have the
-       responsibility to release that memory.  Example for this are
-       the N converter (bug #416288) and the es# converter (bug
-       #501716).
+1. In case of failure, the application cannot know what memory to
+   release; most callers don't even know that they have the
+   responsibility to release that memory.  Example for this are
+   the N converter (bug #416288 [1]_) and the es# converter (bug
+   #501716 [2]_).
 
-    2. Even for successful argument parsing, it is still inconvenient
-       for the caller to be responsible for releasing the memory.  In
-       some cases, this is unnecessarily inefficient.  For example,
-       the es converter copies the conversion result into memory, even
-       though there already is a string object that has the right
-       contents.
+2. Even for successful argument parsing, it is still inconvenient
+   for the caller to be responsible for releasing the memory.  In
+   some cases, this is unnecessarily inefficient.  For example,
+   the es converter copies the conversion result into memory, even
+   though there already is a string object that has the right
+   contents.
 
 
 Proposed solution
+=================
 
-    A new type 'argument tuple' is introduced.  This type derives from
-    tuple, adding an __dict__ member (at tp_dictoffset -4).  Instances
-    of this type might get the following attributes:
+A new type 'argument tuple' is introduced.  This type derives from
+tuple, adding an ``__dict__`` member (at tp_dictoffset -4).  Instances
+of this type might get the following attributes:
 
-       - 'failobjects', a list of objects which need to be deallocated
-         in case of success
+- 'failobjects', a list of objects which need to be deallocated
+  in case of success
 
-       - 'okobjects', a list of object which will be released when the
-         argument tuple is released
+- 'okobjects', a list of object which will be released when the
+  argument tuple is released
 
-    To manage this type, the following functions will be added, and
-    used appropriately in ceval.c and getargs.c:
+To manage this type, the following functions will be added, and
+used appropriately in ``ceval.c`` and ``getargs.c``::
 
-       - PyArgTuple_New(int);
-       - PyArgTuple_AddFailObject(PyObject*, PyObject*);
-       - PyArgTuple_AddFailMemory(PyObject*, void*);
-       - PyArgTuple_AddOkObject(PyObject*, PyObject*);
-       - PyArgTuple_AddOkMemory(PyObject*, void*);
-       - PyArgTuple_ClearFailed(PyObject*);
+- PyArgTuple_New(int);
```

Comment:
Since you've already marked other Py* functions with \`\`Py*\`\`, I'd expect the same for these too.